packages/types/src/saml.ts

```diff
@@ -0,0 +1,46 @@
+export type SamlIdpSlug =
```

Comment:
❓ I would propose to only include the ones we are currently supporting for the time being (okta, google, microsoft, custom) and add the rest in the future

Reply:
Keeping only okta, google, microsoft, custom for now.